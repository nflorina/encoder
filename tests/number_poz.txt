129345678
1
END
