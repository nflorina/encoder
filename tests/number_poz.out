0 0 1
934567812
934567812
